vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -56,6 +58,7 @@ $( document ).ready(function() {
 	$("#cancelSpaceLinkBtn").click(function() {
 		storeX = null;
 		storeY = null;
+		$("#arrow").removeAttr("id");
```

Comment:
if the create procedure is cancelled, the arrow should disappear.

Reply:
Sure. Will remove the arrow upon Cancelling.